manifests/xgboost-operator/base/deployment.yaml

```diff
@@ -1,8 +1,15 @@
-apiVersion: extensions/v1beta1
+apiVersion: extensions/v1
 kind: Deployment
 metadata:
-  name: deployment
+  name: xgboost-operator
+  namespace: xgboost-operator
```

Comment:
i am afriad this is not correct, since you have the namespace on xgboost-operator. but other config is not related to this namespace. this would bring issues. for example , rbac rule is not related to namespace.

Reply:
I will change it. But I will need this eventually right?